source/agg-pipeline-builder/count-pipeline-results.txt

```diff
@@ -0,0 +1,60 @@
+.. _refresh-document-counts:
+.. _count-pipeline-results:
+
+================================
+Count Pipeline Results Documents
+================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+You can view the number of documents outputted by your pipeline with the 
+:guilabel:`count results` button.
+
+About this Task 
+---------------
+
+When you delete or add a document, the :guilabel:`count results` value on the 
+:guilabel:`Aggregations` tab doesn't automatically update to reflect the new 
+document.
+
+Before You Begin 
+----------------
+
+To count result documents, you must first :ref:`create and run 
+<create-agg-pipeline>` your aggregation pipeline.
+
+You cannot count result documents while editing your pipeline.
+
+Steps
+-----
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: Click the count results button
+
+      After you run your pipeline, click :guilabel:`count results`, which 
+      appears under the :guilabel:`Run` button. The button will update with the 
```

Comment:
[question]

Could we make "The button will update with the count..." more specific? Is this saying the Run button will updated with count? I'm a little confused. 

Thank you!

Reply:
Definitely! Updated.